shared/src/main/scala/pl/metastack/metaweb/tag/Body.scala

```diff
@@ -50,97 +50,6 @@ case class Body(attributes: Predef.Map[String, Any] = Predef.Map.empty, children
   def link: scala.Option[String] = attributes.get("link").asInstanceOf[scala.Option[String]]
   def link(value: String): Body = copy(attributes = attributes + ("link" -> value))
   /**
-   * Function to call after the user has printed the document.
-   */
```

Comment:
The problem with these event handlers is that they are typed as strings. We already provide most of them in `Window` and `NodeRefExtensions`.